client/blocks/reader-share/index.jsx

```diff
@@ -64,10 +64,6 @@ const actionMap = {
 };
 
 function buildQuerystringForPost( post ) {
-	const primarySite = sitesList.getPrimary();
```

Comment:
This is no longer needed - we check `canShareToWordPress` before offering WordPress sharing.